src/security/tests/srv_acl_tests.c

```diff
@@ -561,19 +605,19 @@ expect_no_list_access_with_perms(uint64_t acl_perms, uint64_t requested_capas)
 {
 	struct daos_acl		*acl;
 	d_iov_t			cred;
-	struct pool_prop_ugm	ugm;
-	uint32_t		gids[] = { TEST_GID - 1, TEST_GID };
+	struct pool_owner	ownership;
+	static const char	*grps[] = { "wronggroup@", TEST_GROUP };
```

Comment:
(style)  static const char * array should probably be static const char * const

Reply:
I'm inclined to ignore this suggestion unless someone has a good argument for using it - feels like going overboard, and the pointer const syntax appears to be rarely used in the current codebase.